README.md

```diff
@@ -1,2 +1,133 @@
+
+## How to use
+
+The Zeebe C# client boostrap extensions is available via nuget (https://www.nuget.org/packages/zb-client-bootstrap/).
+
+See [examples](./examples) for more information.
+
+## Quick start
+
+All classes which implement `IJobHandler<T>` or `IAsyncJobHandler<T>` are automaticly found and boostrapped when you register this boostrap project with the `IServiceCollection` extension method `BoostrapZeebe`.
+
+The `BoostrapZeebe` method has two parameters:
+
+1. `ZeebeBootstrapOptions` via [Configuration, Action delegate or both](https://docs.microsoft.com/en-us/dotnet/core/extensions/options-library-authors).
+1. An array with assembly filters, only assemblies which start with one of the filters will be scanned for job handlers.
+
+### Appsettings configuration
+
+```csharp
+ConfigureServices((hostContext, services) => {
+    services.BootstrapZeebe(
+        hostContext.Configuration.GetSection("ZeebeBootstrap"),
+        "SimpleExample"
+    );
+})
+```
+
+**appsettings.json**
+
+```json
+{
+    "ZeebeBootstrap": {
+        "Client": {                
+            "GatewayAddress": "0.0.0.0:26500"
+        },
+        "Worker": {
+            "MaxJobsActive": 1,
+            "TimeoutInMilliseconds": 500,
+            "PollIntervalInMilliseconds": 10000,
+            "PollingTimeoutInMilliseconds": 30000
+        }
+    }
+}
+```
+
+### Delegate configuration
+
+```csharp
+ConfigureServices((hostContext, services) => {
+    services.BootstrapZeebe(
+        options => { 
+            options => { 
+                options.Client = new ClientOptions() {
+                    GatewayAddress = "0.0.0.0:26500"
+                };
+                options.Worker = new WorkerOptions() 
+                {
+                    MaxJobsActive = 1,
+                    TimeoutInMilliseconds = 10000,
+                    PollIntervalInMilliseconds = 30000,
+                    PollingTimeoutInMilliseconds = 1000
+                };
+            }
+        },
+        "SimpleExample"
+    );
+})
+```
+
+### Job
+
+The job is an implementation of `AbstractJob`. A job can be configured via optional attributes. Job types must be unique.
+
+```csharp
+[JobType("SimpleJobV2")]
+[WorkerName("SimpleWorker")]
+[MaxJobsActive(2)]
+[Timeout(500)]
+[PollInterval(10000)]
+[PollingTimeout(500)]
+class SimpleJob : AbstractJob
+{
+    public SimpleJob(IJob job) : base(job)
+    { }
+}
+```
+
+### Job handler
+
+The job handler is an implementation of `IJobHandler<T>` or `IAsyncJobHandler<T>`. A job can be configured via optional attributes. Job handlers are automaticly added to the DI container, therefore you can use dependency injection inside the job handlers.
+
+```csharp
+[ServiceLifetime(ServiceLifetime.Singleton)]
+class SimpleJobHandler : IAsyncJobHandler<SimpleJob>
+{
+    public Task HandleJob(IJobClient client, SimpleJob job, CancellationToken cancellationToken)
+    {  
+        return client.NewCompleteJobCommand(job.Key).Send();
```

Comment:
Should cancellationToken nog be given to the Send method?

Reply:
Done